### PR TITLE
Add common user for APS and ACS on Alfresco identity service in helm chart

### PR DESCRIPTION
**Pull request by SohelSaiyed**
* add admin user in “alfresco-realm.json”

Jira-Id: ACTIVITI-2219

### CODE REVIEW

File: alfresco-realm.json [hunk 1]
```diff
@@ -1684,6 +1684,35 @@
       "account" : [ "manage-account", "view-profile" ]
     },
     "groups" : [ "/admin" ]
+  },{
+    "id" : "418b64ff-2ddd-46d5-820d-5979a37cc54b",
```
Comment (rgauss) [inline alfresco-realm.json]: Was this a freshly generated UUID, i.e. not copied and pasted somewhere?

File: alfresco-realm.json [hunk 2]
```diff
@@ -1684,6 +1684,35 @@
...
+    "username" : "admin",
+    "enabled" : true,
+    "totp" : false,
+    "emailVerified" : true,
+    "firstName" : "admin",
+    "lastName" : "admin",
+    "email" : "admin@app.activiti.com",
+    "credentials" : [ {
+      "type" : "password",
+      "hashedSaltedValue" : "+A2UrlK6T33IHVutjQj9k8S8kMco1IMnmCTngEg+PE+2vO4jJScux6wcltsRIYILv5ggcS3PI7tbsynq5u39sQ==",
```
Comment (rgauss) [inline alfresco-realm.json]: Looks like this was copied and pasted from testuser.  I think this needs to be a hashed password of `admin`.

Comment (SohelSaiyed) [reply alfresco-realm.json]: yes the hashed 'password' was copied :-), will update with new value

**Comment by nicdoye** [comment]
lgtm-pineapple-pen

**Pull request merged**